src/main/app/src/components/toteutus/ToteutuksenYhteystiedot.tsx

```diff
@@ -32,6 +32,15 @@ export const ToteutuksenYhteystiedot = ({ oids }: { oids: Array<string> }) => {
     [oppilaitokset]
   );
 
+  const filteredOrganisaatioYhteystiedot = useMemo(
+    () =>
+      oppilaitokset
+        .filter((v) => hasYhteystiedot(v.data.oppilaitoksenOsa?.metadata))
+        .map((v) => v.data.oppilaitoksenOsa.metadata.yhteystiedot)
+        .flat(),
```

Comment:
Jos on vain yksi taso taulukoita taulukon sisällä niin voi kirjottaa:
```suggestion
        .flatMap((v) => v.data.oppilaitoksenOsa.metadata.yhteystiedot)
```